docs/herramientas_test.md

```diff
@@ -0,0 +1,39 @@
+# Herramienta para la elaboración y gestión de tests y pruebas
+
+## Criterios y requisitos
+Al elegir las herramientas adecuadas para la elaboración y gestión de tests y pruebas en Go, es crucial establecer tanto requisitos básicos como criterios de selección detallados. Estos facilitarán la selección de herramientas más eficientes y compatibles con las necesidades de desarrollo.
+
+### Requisitos básicos
+- **Compatibilidad con el lenguaje Go y su toolchain actual**. Las herramientas deben integrarse sin problemas con el entorno de desarrollo Go existente.
+- **Facilidad de configuración**. Deben ser fáciles de configurar y utilizar.
+
+### Criterios de selección
+- **Mantenimiento y actualizaciones regulares**. Las herramientas deben recibir actualizaciones y mantenimiento regulares para garantizar que no quedan obsoletas.
+- **Rendimiento de aserciones y pruebas**. Deben permitir la creación de aserciones rápidas, eficientes y flexibles.
```

Comment:
Si no has testeado explícitamente, o consultado en algún artículo, el rendimiento, no puedes incluirlo. Incluid por favor sólo lo que hayáis considerado explícitamente a la hora de elegir. En tu elección de "Testify" no he visto que hayas mencionado en ningún momento su rendimiento.